.github/workflows/dependency-graph.yaml

```diff
@@ -0,0 +1,21 @@
+# .github/workflows/dependency-graph.yml
+name: Update Dependency Graph
+on:
+  # TODO: remove
```

Comment:
when/why should it be removed?

Reply:
removed it now, the only reason it was there, was to verify that the trigger works, it doesn't make much sense to run it for PRs